examples/config/_common/common.runtime.properties

```diff
@@ -16,16 +16,18 @@
 #
 
 # Extensions (no deep storage model is listed - using local fs for deep storage - not recommended for production)
-druid.extensions.coordinates=["io.druid.extensions:druid-examples","io.druid.extensions:druid-kafka-eight","io.druid.extensions:mysql-metadata-storage"]
```

Comment:
The comments instruct them to uncomment the lines below on line 25 but wouldn't they need to re-add `io.druid.extensions:mysql-metadata-storage` here in `druid.extensions.coordinates` as well?